.github/workflows/deploy_helm_chart.yaml

```diff
@@ -0,0 +1,95 @@
+    runs-on: ubuntu-latest
+    environment: ${{ github.event.inputs.environment }}
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+    steps:
+      - name: Checkout Fairspace repository
+        uses: actions/checkout@v4
+
+      - name: Checkout FNS model and values files
+        uses: actions/checkout@v4
+        with:
+          repository: 'thehyve/fnscloud-demonstrator'
+          path: './fns'
+          token: ${{ secrets.FNS_PAT }}
+
+      - name: Set environment
+        run: |
+          if [ -n "${{ github.event.inputs.environment }}" ]; then
+            ENVIRONMENT="${{ github.event.inputs.environment }}"
+          else
+            ENVIRONMENT="test"
+          fi
+          echo "ENVIRONMENT=$ENVIRONMENT" >> $GITHUB_ENV
+          echo "Starting deployment to the $ENVIRONMENT environment"
+
+      - name: Define Helm Chart version (taken from input or latest snapshot)
+        run: |
+          VERSION=$(cat ./VERSION)-SNAPSHOT
+          if [ -n "${{ github.event.inputs.version }}" ]; then
+            VERSION=${{ github.event.inputs.version }}
+          fi
+          echo "Helm charts version to be deployed: $VERSION"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+      - name: Auth to Google Cloud
+        uses: google-github-actions/auth@v2
+        with:
+          workload_identity_provider: ${{ vars.CICD_WORKLOAD_IDENTITY_PROVIDER }}
+          service_account: ${{ vars.CICD_SERVICE_ACCOUNT_EMAIL }}
+
+      - name: Get GKE credentials to access the cluster
+        uses: google-github-actions/get-gke-credentials@v2
+        with:
+          cluster_name: fns-cloud-f01
+          location: europe-west1
+
+      - name: Setup Helm
+        uses: azure/setup-helm@v4.1.0
+        with:
+          version: ${{ vars.HELM_VERSION }}
+
+      - name: Login with Helm
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | helm registry login ${{ vars.HELM_REGISTRY }} --username ${{ github.repository_owner }} --password-stdin
+
+      - name: Delete StatefulSet (if exists) to force re-deploy
+        run: |
+          # Check if the StatefulSet exists
+          if kubectl get statefulset fairspace --namespace fairspace-${{ env.ENVIRONMENT }} &> /dev/null; then
+            kubectl delete statefulset fairspace --namespace fairspace-${{ env.ENVIRONMENT }} --cascade=orphan
+            echo "StatefulSet fairspace deleted."
+          else
+            echo "StatefulSet fairspace does not exist."
+          fi 
+
+      - name: Deploy Helm chart
+        run: |
+          helm upgrade --install fairspace oci://${{ vars.HELM_REGISTRY }}/fairspace \
+          --version ${{ env.VERSION }} \
+          --namespace=fairspace-${{ env.ENVIRONMENT }} \
+          --set-file saturn.vocabulary=./fns/fairspace/model/vocabulary.ttl \
+          --set-file saturn.views=./fns/fairspace/model/views.yaml \
+          --values ./fns/fairspace/deployment/${{ env.ENVIRONMENT }}/fairspace-test-values.yaml
```

Comment:
I've never used `--values`, we always pass the values path with `-f` flag. 
Does it for sure work? What is the difference between those two?

Reply:
<img width="1083" alt="image" src="https://github.com/thehyve/fairspace/assets/11148506/df3d4de0-d8e2-46cc-9074-b7006d7fd35a">
They are equal, used it for the notation consistency (double-dash) and from hiegher-readability perspective, tested both, no difference found. But let me know if you don't like it, easily can switch to `-f`

Reply:
ok, good to know! I haven't seen it in [the helm values file](https://helm.sh/docs/chart_template_guide/values_files/) documentation,